portfolio/src/main/webapp/Gaming.css

```diff
@@ -0,0 +1,87 @@
+h1 {
```

Comment:
I think best practice is to have lower case file names for css/html files. Can you please rename Gaming* to gaming* and Home* to home*?